pkg/porter/show.go

```diff
@@ -31,7 +31,7 @@ func (so *ShowOptions) Validate(args []string, cxt *context.Context) error {
 	return so.ParseFormat()
 }
 
-// ShowInstallation shows a bundle, or more properly a bundle claim, along with any
+// ShowInstallation shows a bundle installation, along with any
```

Comment:
Thanks these were exactly the type of inaccurate comments that were causing confusion. 👍